nbs/common.base_recurrent.ipynb

```diff
@@ -50,10 +50,12 @@
     "import random\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #186.                stat_exog = windows['static'][:, static_idx]
</pre><p>Why did you remove the 0? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I simplified the wrangling of static variables, and passed forward to the models the responsibility to repeat the statics across the temporal dimension. 

This makes more efficient the `TimeSeriesLoader` and improves its memory usage.
As it is now moving statics of shape `[N,S_in]` rather than shape `[N,T,S_in]`.

The usage on `MLP-based` models is still on the work.